src/compiler/crystal/command/repl.cr

```diff
@@ -1,4 +1,7 @@
 # Implementation of the `crystal repl` command
+{% unless flag?(:without_interpreter) %}
+  require "../interpreter/*"
+{% end %}
```

Comment:
I don't think this particular change is valid. Probably the whole file needs to be skipped
```suggestion
{% skip_file if flag?(:without_interpreter) %}
```
Because otherwise the `repl` method doesn't cause compilation error only because Crystal never looks at it.